db/migrate/20250609182851_make_all_facets_mongo_based.rb

```diff
@@ -1,7 +1,11 @@
 class MakeAllFacetsMongoBased < Mongoid::Migration
   def self.up
     SearchFacet.update_all(is_mongo_based: true)
-    CellMetadatum.where(name: 'organism_age').map(&:set_minmax_by_units!)
+    # Only process CellMetadatum records that have a valid study association
+    CellMetadatum.where(name: 'organism_age').each do |cell_metadatum|
+      next if cell_metadatum.study.nil?
```

Comment:
This shouldn't ever be the case - all records are destroyed on study deletion, though clearly at some point something went sideways locally and you ended up with orphaned data.  I did a quick check on staging & production and we don't have any of these (good), but leaving this check in place is harmless.